tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py36, py37
```

Comment:
Can we support Python 3.5 as well?

Reply:
We should look into the future, not the past, now that Python 3.8 was released :)
But yes, I'll add Python 3.5 :)